org.civicrm.reqangular/src/common/models/instances/session-instance.instance.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-env amd */
+
+define([
+  'common/modules/models-instances',
+  'common/models/instances/instance'
+], function (instances) {
+  'use strict';
+
+  instances.factory('SessionInstance', ['ModelInstance',
```

Comment:
I made a placeholder. It is useless now, but in the past we had placeholders (ex. Absence-tab)